### ISSUE TITLE
Register Portal for ArcGIS WebAdaptor fails: Output of execution: "Portal configuration requires fully qualified domain name in the WebAdaptor URL"

### ISSUE THREAD

**Issue by Biboba** [Esri/arcgis-powershell-dsc#159]
I am testing an ArcGIS Enterprise 10.7 deployment on one machine with 2 web adaptors : one for the ArcGIS Server and one the Portal.
Registering the ArcGIS Server one is successfull but registering the Portal one returns the following message:
```
Output of execution:- Portal configuration requires fully qualified domain name in the WebAdaptor URL
```
Thus it is not registered and it is not consired as an error so at first glance I thought that everything went fine.

Indeed, the hostname is used instead of FQDN:
https://github.com/Esri/arcgis-powershell-dsc/blob/11e0364c92e3cfdb31270d415accabef4a71ce74/Modules/ArcGIS/DSCResources/ArcGIS_WebAdaptor/ArcGIS_WebAdaptor.psm1#L132-L138

It is not a problem for the ArcGIS Server but it fails with Portal.

Thanks

**Comment by shailesh91** [comment]
@Biboba we are not using HostName. If you take a look at the Configure-ArcGIS resource, we do try to get the FQDN of the portal machine before we pass it to web adaptor resource.
https://github.com/Esri/arcgis-powershell-dsc/blob/11e0364c92e3cfdb31270d415accabef4a71ce74/Modules/ArcGIS/Configurations-OnPrem/ArcGISConfigure.ps1#L1163

Can you please attach your JSON config files, Log Files and information of your deployment environment. This will help us repro the said case.

**Comment by Biboba** [comment]
Misconfiguration of the certificate alias property:
solved with #160.
Once again, many thanks @shailesh91
